numerics/poisson_series_basis_body.hpp

```diff
@@ -92,6 +101,18 @@ std::array<Series, dimension * (degree + 1)> PoissonSeriesBasisGenerator<
               Aperiodic(origin)...};
 }
 
+template<typename Series, int dimension, int degree, std::size_t... indices>
+inline std::array<PoissonSeriesSubspace, dimension*(degree + 1)>
+PoissonSeriesBasisGenerator<
+    Series,
+    dimension,
+    degree,
```

Comment:
Put all 3 parameters on a single line, similar to line 98.  In general, `clang-format` doesn't do a great job with these templates...

Reply:
Same formatting comment, see line 169.